checkov/common/images/image_referencer_manager.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from checkov.common.images.image_referencer_provider import Provider
+
+from typing import Any, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from checkov.common.images.image_referencer import Image
+
+
+class ImageReferencerManager:
```

Comment:
```suggestion
class WorkflowImageReferencerManager:
```
this IR manager is specific to workflow files, so let's also give him an appropriate name 🙂